### DISCUSSION TITLE
NVENC 10bit on Handbrake 1.5.1

### DISCUSSION THREAD

**Discussion by gotterfield** [RandomEngy/VidCoder#1026] [General]
NVENC 10bit encoding available on Handbrake 1.5.1, it doesn't appear on the latest Vidcoder.

**Comment by RandomEngy** [comment]
Is it? It doesn't show up for me in HandBrake 1.5.1.

![image](https://user-images.githubusercontent.com/9450861/171546770-7d11f947-dede-4cc4-88ac-7c0a41e8db2c.png)
